src/app/components/card/CardButton.vue

```diff
@@ -20,10 +20,16 @@
           </span>
         </div>
         <div class="opacity-60">
-          <slot v-if="$slots.iconSecondary" name="iconSecondary" />
+          <slot v-if="$slots.iconS" name="iconS" />
```

Comment:
As always, let's not start using abbreviations (except there is no way around it).